web/annotation_editor_layer_builder.css

```
@@ -270,6 +262,14 @@
       }
 
       > * {
+        height: var(--editor-toolbar-height);
+      }
+
+      > :not(:is(.divider)) {
```

Isn't the `:is`-part redundant here?
```suggestion
      > :not(.divider) {
```